app/controllers/users_controller.rb

```diff
@@ -37,7 +37,12 @@ def update
     end
   end
 
-  def destroy; end
+  def destroy
+    @user = User.find(params[:id])
+    @user.destroy
+
```

Comment:
Pode fazer direto isso:

```suggestion
    User.destroy(params[:id])
```